capstone/src/main/webapp/youtubeTypescript.js

```diff
@@ -68,8 +68,8 @@ var YOUTUBE_CATEGORIES = {
     'Sports': 17
```

Comment:
Once the typescript PR is in, let's make sure this file is deleted, that way in the github diff we only see 1 set of changes (in the TypeScript file)

Reply:
addressed in "add gitignore and delete cache"